draft-reddy-tls-composite-mldsa.md

```diff
@@ -131,6 +136,13 @@ defined in {{I-D.ietf-lamps-pq-composite-sigs}}.
 
 The schemes defined in this document MUST NOT be used in TLS 1.2 {{RFC5246}}. A peer that receives ServerKeyExchange or CertificateVerify message in a TLS 1.2 connection with schemes defined in this document MUST abort the connection with an illegal_parameter alert.
 
+# Signature Algorithm Restrictions
+
+TLS 1.3 removed support for RSASSA-PKCS1-v1_5 {{RFC8017}} in CertificateVerify messages, opting for RSASSA-PSS instead. Similarly, this document restricts the use of the composite signature algorithms mldsa44_rsa_pkcs1_sha256 and mldsa65_rsa_pkcs1_sha384 to the "signature_algorithms_cert" extension. These composite signature algorithms MUST NOT be used with the "signature_algorithms" extension.
```

Comment:
1. old names used (not `mldsa44_rsa2048_pkcs1_sha256` nor `mldsa65_rsa3072_pkcs1_sha256`)
2. `mldsa65_rsa4096_pkcs1_sha384` missing

Reply:
Good catch, fixed.